students/pkoleyni/lesson04/mailroom_2.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3
+import datetime
+
+donor_list_dic = {'pooria koleyni': {'donation': [130]},
+                  'john adams': {'donation': [200, 340, 560]},
+                  'steve knot': {'donation': [50, 130]},
+                  'meru cheng': {'donation': [150, 430]},
+                  'maysam razm': {'donation': [25, 25]}
+                  }
+
+
+def list_donors():
+    """
+    Return list of name of donors by getting keys for the donor_list_dic dictionary
+    :return:
+    """
+    donor_list = donor_list_dic.keys()
+    return donor_list
```

Comment:
I would recommend to not continue using this function.

For pythonic style, use `.items()` and loop over keys and values